### PR TITLE
Create

### PR DESCRIPTION
**Pull request by celinevo**

### CODE REVIEW

File: src/components/CreepypastaForm.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+  return (
+    <Form
+      aria-label="Create a creepypasta"
+      onSubmit={handleSubmit(creepypasta => onSubmit(creepypasta))}
+    >
+      <Field>
+        <Label htmlFor="title">Title:</Label>
+        <Input
+          {...register('title', {
```
Comment (J-Massmann) [inline src/components/CreepypastaForm.js:43]: you could think about adding `autoFocus` to your first input, so the user will be directed straight to the form

File: src/components/CreepypastaForm.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  function onSubmit(creepypasta) {
+    handleCreateCreepypasta({
+      id: nanoid(),
+      title: creepypasta.title,
+      text: creepypasta.text,
+      wordcount: wordCount,
+      image: creepypasta.image,
+      isSpookmarked: false,
+    });
+    navigate('/');
```
Comment (J-Massmann) [inline src/components/CreepypastaForm.js]: You can think about adding a `Modal` (it's like a Pop-up/Submitmessage for the user)
It is quite easy to implement, took me like 15 min :)

File: src/components/CreepypastaForm.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+      aria-label="Create a creepypasta"
+      onSubmit={handleSubmit(creepypasta => onSubmit(creepypasta))}
+    >
+      <Field>
+        <Label htmlFor="title">Title:</Label>
+        <Input
+          {...register('title', {
+            required: 'You need a spooky title!',
+            maxLength: {
+              value: 25,
```
Comment (J-Massmann) [inline src/components/CreepypastaForm.js:53]: Think about adding a counter for the user, so he/she knows how many characters he is allowed to use.
Another idea would be to add `useForm(mode:all)` , which includes the mode `onChange`, so the error message will appear when the user puts in more than 25 characters

File: src/components/CreepypastaForm.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        </Wordcount>
+      </Field>
+      <Field>
+        <Label htmlFor="image">Image:</Label>
+        <Input
+          {...register('image', { required: 'You need a creepy image!' })}
+          id="image"
+          placeholder=" Please use an URL"
+          type="url"
+          pattern="https://"
```
Comment (J-Massmann) [inline src/components/CreepypastaForm.js]: You could leave a little note for the user which format the image url needs to be.
I tried to upload an image to test it, but I failed 😂

File: src/components/CreepypastaForm.spec.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    const imageInput = screen.getByLabelText(/image/i);
+    const textTextarea = screen.getByLabelText(/text/i);
+    const saveButton = screen.getByRole('button');
+
+    expect(titleInput).toBeInTheDocument();
+    expect(imageInput).toBeInTheDocument();
+    expect(textTextarea).toBeInTheDocument();
+    expect(saveButton).toBeInTheDocument();
+  });
+});
```
Comment (J-Massmann) [inline src/components/CreepypastaForm.spec.js:23]: add another test to check if the function `handlecreepypasta` is called when the button is clicked

**Review by J-Massmann — COMMENTED** [review]
All in all you code looks good, just some hints here and there 🙂

**Review by ttaannjjaa — APPROVED** [review]
looks very good to me

**Pull request merged**